### PR TITLE
Update version

### PR DESCRIPTION
**Pull request by mohammedDehairy**
Update version.

### CODE REVIEW

File: Adyen.podspec [hunk 1]
```diff
@@ -11,8 +11,8 @@ Pod::Spec.new do |s|
   s.author = { 'Adyen' => 'support@adyen.com' }
   s.source = { :git => 'https://github.com/Adyen/adyen-ios.git', :tag => "#{s.version}" }
   s.platform = :ios
-  s.ios.deployment_target = '11.0'
-  s.swift_version = '5.1'
+  s.ios.deployment_target = '4.7.41.0'
+  s.swift_version = '5.4.7.4'
```
Comment (erenbesel) [inline Adyen.podspec:15]: this does not seem right :D

File: Adyen.podspec [hunk 2]
```diff
@@ -31,7 +31,7 @@ Pod::Spec.new do |s|
     plugin.source_files = 'AdyenWeChatPay/**/*.swift'
     plugin.dependency 'Adyen/Core'
     plugin.dependency 'Adyen/Actions'
-    plugin.dependency 'AdyenWeChatPayInternal', '2.1.0'
+    plugin.dependency 'AdyenWeChatPayInternal', '2.4.7.4.0'
```
Comment (erenbesel) [inline Adyen.podspec:34]: This as well

File: Adyen.podspec [hunk 3]
```diff
@@ -77,7 +77,7 @@ Pod::Spec.new do |s|
   s.subspec 'Core' do |plugin|
     plugin.source_files = 'Adyen/**/*.swift'
     plugin.exclude_files = 'Adyen/**/BundleSPMExtension.swift'
-    plugin.dependency 'AdyenNetworking', '1.0.0'
+    plugin.dependency 'AdyenNetworking', '4.7.4.0.0'
```
Comment (erenbesel) [inline Adyen.podspec:80]: Also this

File: Adyen.podspec (hunk 1 above)
Comment (neelSharma12) [reply Adyen.podspec:15]: yes. I was testing the job which updates the version 😸

**Comment by neelSharma12** [comment]
This PR was raised from the update version job for testing purpose.

**Pull request closed without merging**